### DISCUSSION TITLE
Split numbers with comma

### DISCUSSION THREAD

**Discussion by CFarhad** [mantinedev/mantine#3636] [Q&A]
how i can split numbers with the comma? is this possible in `Text` component

**Comment by rtivital** [comment]
Sure: `<Text>1,000,000</Text>`

**Comment by CFarhad** [comment]
@rtivital I know that. I want it do it automatically

**Reply by rtivital** [reply]
You can use Intl for that or install a library that formats numbers. It is outside of components library scope
